runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/MetricTrackingWindmillServerStub.java

```diff
@@ -239,19 +240,23 @@ public Windmill.GlobalData getSideInputData(Windmill.GlobalDataRequest request)
   }
 
   /** Tells windmill processing is ongoing for the given keys. */
```

Comment:
Can you update the comment? Unclear what heartbeats is

Reply:
Hopefully that's better.